keras_cv/models/object_detection/retina_net/retina_net_presets.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""RetinaNet Task presets."""
+from keras_cv.models.backbones.resnet_v1 import resnet_v1_backbone_presets
+
+retina_net_presets = {
+    "resnet50_pascal_voc": {
+        "metadata": {
+            "description": (
+                "ResNet classifier with 50 layers where the batch "
+                "normalization and ReLU activation precede the convolution "
+                "layers (v2 style). Trained on Imagenet 2012 classification "
+                "task."
+            ),
+        },
+        "config": {
+            "backbone": resnet_v1_backbone_presets.backbone_presets["resnet50"],
+            # Should be 21
+            "num_classes": 20,
+        },
+        "weights_url": "https://storage.googleapis.com/keras-cv/models/retinanet/pascal_voc/resnet50.weights.h5",
```

Comment:
Why `.weights`?

Reply:
there is a pretty long thread about this in the main chat;
TLDR: This is the new Keras model SavedFormat, and when you're using a sub-class model including any sort of floating layers/floating weights this will behave much more consistently.

This is also meant to be the new Keras model format going forwards

Reply:
i can't omit it for now.  I couldn't get the model to load without the new format.

Reply:
URLs are arbitrary strings, so it doesn't make any user-facing difference. Long-term all files should standardize on the new format. This is low priority though since it doesn't really change anything.

Reply:
Sorry I actually misunderstood this change. This only affects the name of the file in the local cache.